### PR TITLE
pinned ios sdk dependency

### LINKED ISSUE #1074: iOS SDK App not compiling
[xcode-errors.txt](https://github.com/openwhisk/openwhisk/files/422823/xcode-errors.txt)
## Environment details:

* Mac OS El Capitan
* XCode 7.3.1
* WSK CLI version `2016-08-10T15:49:14+00:00`

## Steps to reproduce the issue:

1.   Run `wsk sdk install iOS`
2.   Unzip `OpenWhiskIOSStarterApp.zip`
3.   Run `pod install`
4.   Open with XCode 7.3.1 `OpenWhiskStarterApp.xcworkspace`
5.   Build and Run target/scheme `OpenWhiskStarterApp`

## Provide the expected results and outputs:

```
No warnings and erros, app should compile clean
```

## Provide the actual results and outputs:

```
/Users/csantana23/Documents/dev/whisk/demos/ios0817/OpenWhiskStarterApp/ViewController.swift:80:49: error: tuple pattern has the wrong length for tuple type '(description: String, statusCode: Int)'
                if case let WhiskError.HTTPError(description, statusCode, _) = error {
```

@paulcastro FYI.

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #1074

### CODE REVIEW

**Comment by csantanapr** [comment]
@paulcastro please review.
Once review I will move it to ready and have someone else merge it
take into account we can't merge until this one it's merge and tag 0.1.7 created https://github.com/openwhisk/openwhisk-client-swift/pull/18

**Comment by csantanapr** [comment]
I will work on updating the docs to reflect the changes

File: mobile/iOS/starterapp/Podfile [hunk 1]
```diff
@@ -1,8 +1,7 @@
-source 'https://github.com/openwhisk/openwhisk-podspecs.git'
-
+install! 'cocoapods', :deterministic_uuids => false
```
Comment (paulcastro) [inline mobile/iOS/starterapp/Podfile]: Per  @AnthonyOliveri, this line is optional.  It does get rid of some annoying warnings so I'm OK keeping it.

**Comment by paulcastro** [comment]
LGTM, do we need to rebase?  pbxproj files etc have changed.

**Comment by csantanapr** [comment]
rebase, added docs
please review @paulcastro @rabbah

**Comment by paulcastro** [comment]
LGTM

**Comment by csantanapr** [comment]
can you merge this one?

**Comment by csantanapr** [comment]
going to rebase, can't tell why Travis is mad this commits.

**Comment by csantanapr** [comment]
Travis happy now please merge @bjustin-ibm

**Comment by bjustin-ibm** [comment]
Rebased and merged as fc66e6d1208373f2052433123d1d4cdcbdc14694

**Pull request closed without merging**